Mobile Buy SDK/Mobile Buy SDK/Utils/BUYModelManager+ApplePay.h

```diff
@@ -0,0 +1,33 @@
+//
+//  BUYModelManager+ApplePay.h
+//  Mobile Buy SDK
+//
+//  Created by Gabriel O'Flaherty-Chan on 2016-05-20.
+//  Copyright © 2016 Shopify Inc. All rights reserved.
+//
+
+#import <Buy/BUYModelManager.h>
+@class BUYAddress;
```

Comment:
Nit: Can we not mix forward declarations with import statements? `@class` should be after all imports.